TTLocalizer: Running in language: english
from toontown.toonbase.TTLocalizerEnglish import *
OTPLocalizer: Running in language: english
from otp.otpbase.OTPLocalizerEnglish import *
Preloading suits...
Preloading Toon heads...
Preloading avatars...
{401000001: <toontown.distributed.ToontownDistrict.ToontownDistrict instance at 0x1F4F31C0>}
__handleAvatarChooserDone: 100000003, 'Little Zippy', ('rsl', 'ls', 'l', 'm', 19, 0, 19, 19, 10, 9, 0, 9, 7, 0), 1
2
Reading DNA file... phase_4/dna/storage.pdna
Reading DNA file... phase_3.5/dna/storage_interior.pdna
Reading DNA file... phase_4/dna/storage_TT.pdna
Reading DNA file... phase_4/dna/storage_TT_sz.pdna
Reading DNA file... phase_4/dna/toontown_central_sz.pdna
Exception occurred while processing Func-catch-74 of Sequence-97:
Sequence-97:
  0.000 Sequence-97 {
  0.000   *Func-catch-55
  0.000   Wait dur 0.05
  0.050   *Func-catch-56
  0.050   Wait dur 0.05
  0.100   *Func-catch-57
  0.100   Wait dur 0.05
  0.150   *Func-catch-58
  0.150   Wait dur 0.05
  0.200   *Func-catch-59
  0.200   Wait dur 0.05
  0.250   *Func-catch-60
  0.250   Wait dur 0.05
  0.300   *Func-catch-61
  0.300   Wait dur 0.05
  0.350   *Func-catch-62
  0.350   Wait dur 0.05
  0.400   *Func-catch-63
  0.400   Wait dur 0.05
  0.450   *Func-catch-64
  0.450   Wait dur 0.05
  0.500   *Func-catch-65
  0.500   Wait dur 0.05
  0.550   *Func-catch-66
  0.550   Wait dur 0.05
  0.600   *Func-catch-67
  0.600   Wait dur 0.05
  0.650   *Func-catch-68
  0.650   Wait dur 0.05
  0.700   *Func-catch-69
  0.700   Wait dur 0.05
  0.750   *Func-catch-70
  0.750   Wait dur 0.05
  0.800   *Func-catch-71
  0.800   Wait dur 0.05
  0.850   *Func-catch-72
  0.850   Wait dur 0.05
  0.900   *Func-catch-73
  0.900   Wait dur 0.05
  0.950   *Func-catch-74
  0.950   Wait dur 0.05
  1.000   *Func-catch-75
  1.000   Wait dur 0.05
  1.050   *Func-catch-76
  1.050   Wait dur 0.05
  1.100   *Func-catch-77
  1.100   Wait dur 0.05
  1.150   Wait dur 1e-005
  1.150   *Func-sell-78
  1.150   Wait dur 0.75
  1.900 }

Traceback (most recent call last):
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\showbase\ShowBase.py", line 1801, in __ivalLoop
    IntervalManager.ivalMgr.step()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 96, in step
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 124, in __doPythonCallbacks
    self.ivals[index].privPostEvent()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 518, in privPostEvent
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 494, in __doPythonCallbacks
    ival.privDoEvent(t, eventType)
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\Interval.py", line 207, in privDoEvent
    self.privInstant()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\FunctionInterval.py", line 88, in privInstant
    self.function(*self.extraArgs, **self.kw)
  File "<string>", line 26, in catch
  File "toontown\fishing\DistributedFishingPond.py", line 78, in d_hitTarget
    self.localToonSpot.hitTarget()
AttributeError: 'NoneType' object has no attribute 'hitTarget'
Traceback (most recent call last):
  File "otp\otpbase\OTPBase.py", line 236, in run
    taskMgr.run()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\task\Task.py", line 502, in run
    self.step()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\task\Task.py", line 460, in step
    self.mgr.poll()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\showbase\ShowBase.py", line 1801, in __ivalLoop
    IntervalManager.ivalMgr.step()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 96, in step
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\IntervalManager.py", line 124, in __doPythonCallbacks
    self.ivals[index].privPostEvent()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 518, in privPostEvent
    self.__doPythonCallbacks()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\MetaInterval.py", line 494, in __doPythonCallbacks
    ival.privDoEvent(t, eventType)
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\Interval.py", line 207, in privDoEvent
    self.privInstant()
  File "D:\Documents\ToontownSourcecode\Panda3D-1.9.0\direct\interval\FunctionInterval.py", line 88, in privInstant
    self.function(*self.extraArgs, **self.kw)
  File "<string>", line 26, in catch
  File "toontown\fishing\DistributedFishingPond.py", line 78, in d_hitTarget
    self.localToonSpot.hitTarget()
AttributeError: 'NoneType' object has no attribute 'hitTarget'
